metricbeat/docs/modules/beat.asciidoc

```diff
@@ -5,16 +5,21 @@ This file is generated! See scripts/mage/docs_collector.go
 [[metricbeat-module-beat]]
 == Beat module
 
-The Beat module contains a minimal set of metrics to enable monitoring of any Beat or other software based on libbeat across
-multiple versions. To monitor more Beat metrics, use our {stack}
-{monitor-features}.
-
-The default metricsets are `state` and `stats`.
+The `beat` module collects metrics about any Beat or other software based on libbeat.
 
 [float]
 === Compatibility
 
-The Beat module works with Beats 7.3.0 and later.
+The `beat` module work with {beats} 7.3.0 and later.
+
+[float]
+=== Usage for Stack Monitoring
+
+The `beat` module can be used to collect metrics shown in our {stack} {monitor-features}
+UI in {kib}. To enable this usage, set `xpack.enabled: true` and remove any `metricsets`
+from the module's configuration. Alternatively, run `metricbeat modules disable beat` and
+metricbeat modules enable beat-xpack`.
+
```

Comment:
@jsoriano Based on your feedback, I updated the docs for the `beat` module. WDYT? @dedemorton I'd like your feedback as well.

If/when we're happy with the `beat` module docs, I'll make similar changes to other stack modules' docs in this PR.

Reply:
LGTM.